src/content/NewsContent.tsx

```diff
@@ -16,6 +16,14 @@
 import { PostDescription } from '../theme/types';
 
 const news: PostDescription[] = [
+  {
+    title: 'November 2022 News',
+    text: "Check out our interview article with Prof. Wil van der Aalst, the documentation of the R package and more in this November's news.",
+    cover: 'https://miro.medium.com/max/1000/0*z93yiiu8llcEuEOM',
```

Comment:
probably too large, waiting for the surge preview to be available

Reply:
142 KB